packages/frontend/src/components/common/Dashboard/DashboardHeader.tsx

```diff
@@ -155,6 +163,18 @@ const DashboardHeader = ({
             projectUuid,
         }),
     );
+    const isPromoteChartsEnabled = useFeatureFlagEnabled(
+        FeatureFlags.PromoteCharts,
+    );
+    const userCanPromoteDashboard =
```

Comment:
we only check if they have the right role, we don't check for underlying space/chart permissions. In the future we could make a new endpoint only to return a more detailed  permission check.